source/documentation/values/mixins.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Mixins
+---
+
+{% compatibility 'dart: "1.69.0"', 'libsass: false', 'ruby: false' %}{% endcompatibility %}
+
+[Mixins] can also be values! You can't directly write a mixin as a value, but
+you can pass a mixin's name to the [`meta.get-mixin()` function] to get it as a
+value. Once you have a mixin value, you can pass it to the [`meta.apply()`
+mixin] to call it. This is for libraries to be extensible in complex and
```

Comment:
```suggestion
function] to call it. This is for libraries to be extensible in complex and
```
you may need to reflow

Reply:
[`meta.apply`](https://github.com/sass/sass/blob/main/accepted/first-class-mixins.d.ts.md#metaapply) _is_ a mixin, so I believe this case is correct

Reply:
I see, then that means the link below is wrong, I'll resolve this one and comment there